decidim-core/db/migrate/20200327082257_migrate_newsletters_to_templates.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class MigrateNewslettersToTemplates < ActiveRecord::Migration[5.2]
+  class ContentBlock < ApplicationRecord
+    self.table_name = :decidim_content_blocks
+  end
+
+  class Newsletter < ApplicationRecord
```

Comment:
```suggestion
  class ::Newsletter < ApplicationRecord
```

Reply:
I've checked all other migrations defining classes and none uses this syntax 😕